R/readNGWMNdata.R

```diff
@@ -5,9 +5,12 @@
 #' @param \dots Other parameters to supply, namely \code{siteNumbers} or \code{bbox}
 #' @param asDateTime logical if \code{TRUE}, will convert times to POSIXct format.  Currently defaults to 
 #' \code{FALSE} since time zone information is not included.  
-#' @param tz character to set timezone attribute of datetime. Default is an empty quote, which converts the 
-#' datetimes to UTC (properly accounting for daylight savings times based on the data's provided time zone offset).
-#' Accepts all values from \code{OlsonNames()}.
+#' @param tz character to set timezone attribute of dateTime. Default is "UTC", and converts the 
+#' date times to UTC, properly accounting for daylight savings times based on the data's provided tz_cd column.
```

Comment:
Is this slightly misleading, since there isn't a tz_cd column?  Time zones are just included with the dateTime